libsolidity/codegen/ir/IRGenerationContext.h

```diff
@@ -152,6 +152,14 @@ class IRGenerationContext
 	bool inlineAssemblySeen() const { return m_inlineAssemblySeen; }
 	void setInlineAssemblySeen() { m_inlineAssemblySeen = true; }
 
+	/// @returns the runtime ID to be used for the function in the dispatch routine
+	/// and for internal function pointers.
+	/// @param _requirePresent if false, generates a new ID if not yet done.
+	uint64_t internalFunctionID(FunctionDefinition const& _function, bool _requirePresent);
+	/// Copies the internal function IDs from the @a _other. For use in transferring
+	/// function IDs from constructor code to deployed code.
+	void copyFunctionIDs(IRGenerationContext const& _other);
```

Comment:
"Import" would indicate the direction better than "copy". Especially at the call site where you don't necessarily see that the parameter is `const`.
```suggestion
	void importFunctionIDs(IRGenerationContext const& _other);
```

Reply:
To me, "import" has the additional meaning that nothing is really copied over, but we rather just keep a reference.
I don't understand what this has to do with the parameter being `const` - is it about the function ids from `_other` not being modified? Or is it about the direction of the copy?

Reply:
Or sorry, you even wrote "direction"...

Reply:
In that case, I would prefer `copyFunctionIDsFrom`

Reply:
> I don't understand what this has to do with the parameter being const - is it about the function ids from _other not being modified?

Yeah. If it's const then it's clear that we must be copying from it, not to it.

Anyway. The new name is fine with me.